agent/engine/task_manager.go

```diff
@@ -1464,6 +1466,12 @@ func (mtask *managedTask) cleanupTask(taskStoppedDuration time.Duration) {
 	mtask.engine.sweepTask(mtask.Task)
 	mtask.engine.deleteTask(mtask.Task)
 
+	// Remove TaskExecutionCredentials from credentialsManager
+	if taskExecutionCredentialsID != "" {
+		logger.Info("Cleaning up task's execution credentials")
```

Comment:
minor: can we also log task ID here?

Reply:
Added taskArn here following log format in this file 😀